persistence-api/src/main/java/io/stargate/db/query/builder/QueryBuilderImpl.java

```diff
@@ -665,6 +666,19 @@ public void limit(Integer limit) {
     }
   }
 
+  @DSLAction
+  public void limitPerPartition() {
```

Comment:
We should stick to the CQL terminology and name this `perPartitionLimit()`.

Regardless of which name is better, we can't change CQL and I think it's more important to be consistent across the board (for example when people look for the option with autocomplete, etc).

Reply:
updated

Reply:
I renamed field names to follow the `perPartitionLimit` pattern in the latest commit, which also fixes a bug in parameter passing to `Bound`.